src/hooks/user/useEmail.tsx

```diff
@@ -0,0 +1,37 @@
+import { PaginationResponse, EmailDomain } from '@@types/index';
+import { defaultPaginationValue } from '@@types/PaginationType';
+import useApi from '@hooks/useApi';
+import { useSearchParams } from 'react-router-dom';
+
+interface FetchAllEmailsParamsType {
```

Comment:
```suggestion
interface FetchAllEmailDomainsParamsType {
```
super admin도 이렇게 적혀있던거 같은데 통일하는게 좋아보이네요

Reply:
```js
interface FetchAllEmailsParamsType {
  page: number;
  size: number;
  name?: string;
}

interface AddEmailDomainParamsType {
  name: string;
  domain: string;
}
```

이게 super admin 코드인데, 여기서의 emails도 그냥 emailDomains로 바꿀까요? ( + 기존 emailis 라고 쓰여진 모든 부분들 포함)

Reply:
아 add만 저렇게 되어있었구나... useEmail(훅 이름)은 유지하시고

나머지 도메인과 관련된 부분은 emailDomain 통일시켜주시죠!

Reply:
그러면 `Email, EmailContent, SuperAdminEmailContent, SuperAdminEmail` 해당 컴포넌트들도 모두 emailDomain으로 이름 바꾸나요? 아니면 컴포넌트는 유지하나요?

Reply:
개인적으로 바꾸는게 맞다고 생각합니다.

나중에 진짜 email을 조회하는 경우가 생길 수도 있으니깐요

Reply:
넵 알겠습니다~